app/controllers/issues_controller.rb

```diff
@@ -32,6 +32,11 @@ def show
     @first_evidence  = Evidence.where(node: @first_node, issue: @issue)
 
     load_conflicting_revisions(@issue)
+
+    @subscription = Subscription.find_by(
```

Comment:
On a purist note I think it's best to use `find_by!` instead of `find_by` if we're not excepting that it will ever return `nil`. That way if something goes wrong and it *does* return `nil`, the error gets caught right here instead of bubbling down to the view or wherever where it can be much harder to debug.